scripts/us_bea/states_gdp/import_industry_data_and_gen_mcf.py

```diff
@@ -0,0 +1,168 @@
+            """Filters out aggregate NAICS codes and assigns them their Data
+            Commons codes.
+            """
+            if naics_code == "321,327-339":
+                return "JOLTS_320000"
+            if naics_code == "311-316,322-326":
+                return "JOLTS_340000"
+            return naics_code.replace("-", "_")
+        df['NAICS'] = df['IndustryClassification'].apply(convert_industry_class)
+
+        self.df = df.drop(["GeoFIPS","IndustryClassification"], axis=1)
+
+    def download_data(self):
+        """Downloads zip, extracts the desired CSV, and puts it into a DF."""
+        # Open zip file from link.
+        resp = urlopen(self.ZIP_LINK)
+
+        # Read the file, interpret it as bytes, and create a ZipFile instance
+        # from it for easy handling.
+        zip_file = zipfile.ZipFile(io.BytesIO(resp.read()))
+
+        # Open the specific desired file (CSV) from the folder, and decode it.
+        # This results in a string representation of the file. Interpret that
+        # as a CSV, and read it into a DF.
+        data = zip_file.open(self.FILE).read().decode('utf-8')
+        data = list(csv.reader(data.splitlines()))
+        df = pd.DataFrame(data[1:], columns=data[0])
+        return df
+
+    def save_csv(self, prefix='states_industry_gdp'):
+        """Splits instance data frame into multiple CSV files, one per industry
+        code. This facilitates TMCF design.
+        """
+        for naics_code, naics_df in self.df.groupby("NAICS"):
+            naics_df.to_csv(prefix + f"_{naics_code}.csv")
+
+    def generate_MCF_TMCF(self):
+        """Generates MCF StatVars for each industry code."""
+        mcf_temp = ('Node: dcid:USSateQuarterlyIndustryGDP_NAICS_{naics}\n'
+                    'typeOf: dcs:StatisticalVariable\n'
+                    'populationType: dcs:EconomicActivity\n'
+                    'activitySource: dcs:GrossDomesticProduction\n'
+                    'measuredProperty: dcs:amount\n'
+                    'measurementQualifier: dcs:Nominal\n'
+                    'naics: dcid:NAICS/{naics}\n\n')
+        tmcf_temp = ('Node: E:states_industry_gdp_{naics}->E1\n'
+                     'typeOf: dcs:StatVarObservation\n'
+                     'variableMeasured: dcs:USSateQuarterlyIndustryGDP_NAICS_{naics}\n'
+                     'observationAbout: C:states_industry_gdp_{naics}->GeoId\n'
+                     'observationDate: C:states_industry_gdp_{naics}->Quarter\n'
+                     'observationPeriod: "P3M"\n'
+                     'value: C:states_industry_gdp_{naics}->value\n'
+                     'unit: "Current USD"\n\n')
+
+        with open('states_gdp_industry_statvars.mcf', 'w') as mcf_f, \
+             open('states_industry_gdp.tmcf', 'w') as tmcf_f:
+            for naics_code in self.df['NAICS'].unique():
+                mcf_f.write(mcf_temp.format(naics=naics_code))
+                tmcf_f.write(tmcf_temp.format(naics=naics_code))
+
+def main(argv):
```

Comment:
https://google.github.io/styleguide/pyguide.html#214-decision hints we might be able to write this as

```
def main(_):
```

Reply:
Done.